### PR TITLE
[FR] Fixes before merge

### PR DESCRIPTION
**Pull request by sestegra**
### Describe your changes

- [X] Fix frontmatter that aren't aligned with `main` branch
- [x] Fix internal links to point on French version
- [x] Fix dead links
- [x] Add missing links
- [x] Fix general errors
- [x] Localize tags

### Related issue number or link (ex: `resolves #issue-number`)

N/A

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/abstraction.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ category: Property
 tags: ["fundamental", "", ""]
```
Comment (jihoon-seo) [inline content/fr/abstraction.md:5]: How about localizing tags? 😊
e.g., https://glossary.cncf.io/pt-br/tags/

**Review by jihoon-seo — COMMENTED** [review]
@sestegra Thank for your hard work!

File: content/fr/abstraction.md (hunk 1 above)
Comment (sestegra) [reply content/fr/abstraction.md:5]: Indeed, I put the PR back to Draft to localize the tags.

Comment (sestegra) [reply content/fr/abstraction.md]: I just pushed a new commit to localize the tags

**Review by Krast76 — APPROVED** [review]

**Review by huats — APPROVED** [review]
Thanks a lot for such work ! it looks great !

**Pull request merged**